internal/pkg/service/common/etcdop/op/txn_test.go

```diff
@@ -319,22 +323,6 @@ get bar bar
 txn OnSucceeded
 txn OnResult: succeeded
 `,
-			ExpectedTxnResult: txnResult{
-				Succeeded: true,
-				Results: []any{
-					// Results from the Then branch
-					NoResult{},
-					NoResult{},
-					&KeyValue{
-						Key:   []byte("key/foo"),
-						Value: []byte("foo"),
-					},
-					&KeyValue{
-						Key:   []byte("key/bar"),
-						Value: []byte("bar"),
-					},
-				},
-			},
```

Comment:
Updated tests, without sub-results.

In complex transactions, it was problem to know under which index the result should be searched.